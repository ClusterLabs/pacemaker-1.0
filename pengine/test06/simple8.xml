<?xml version="1.0"?>
<!DOCTYPE cib SYSTEM "crm-1.0.dtd">
<!-- shutdown -->
<cib num_updates="1" epoch="1" admin_epoch="0">
  <configuration>
    <crm_config><cluster_property_set id="cib-bootstrap-options"><attributes>
      <nvpair id="no_quorum_policy" name="no_quorum_policy" value="ignore"/>
      <nvpair id="symmetric_cluster" name="symmetric_cluster" value="true"/>
      <nvpair id="default_resource_stickiness" name="default_resource_stickiness" value="100"/>
    </attributes></cluster_property_set></crm_config>
    <nodes>
      <node id="node1" uname="node1" type="member"/>
      <node id="node2" uname="node2" type="member"/>
    </nodes>
    <resources>
      <primitive id="rsc3" class="heartbeat" type="apache" />
      <primitive id="rsc4" class="heartbeat" type="apache" />
      <group id="foo">
	<primitive id="rsc1" class="heartbeat" type="apache"/>
      </group>
      <group id="bar">
	<primitive id="rsc2" class="heartbeat" type="apache" />
      </group>
    </resources>
    <constraints>
    </constraints>
  </configuration>
  <status>
    <node_state id="node1" ha="active" uname="node1" crmd="online" join="member" expected="member" in_ccm="true">
      <lrm id="node1">
        <lrm_resources>
          <lrm_resource id="foo:rsc1" class="heartbeat" type="apache">
            <lrm_rsc_op id="foo:rsc1_start_0" operation="start" interval="0" op_status="0" rc_code="0" call_id="1" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" crm_feature_set="1.0.6" transition_magic=""/>
          </lrm_resource>
          <lrm_resource id="rsc3" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc3_start_0" operation="start" interval="0" op_status="0" rc_code="0" call_id="1" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" crm_feature_set="1.0.6" transition_magic=""/>
          </lrm_resource>
          <lrm_resource id="bar:rsc2" class="heartbeat" type="apache">
            <lrm_rsc_op id="bar:rsc2_start_0" operation="start" interval="0" op_status="0" rc_code="0" call_id="1" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" crm_feature_set="1.0.6" transition_magic=""/>
          </lrm_resource>
          <lrm_resource id="rsc4" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc4_start_0" operation="start" interval="0" op_status="0" rc_code="0" call_id="1" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" crm_feature_set="1.0.6" transition_magic=""/>
          </lrm_resource>
        </lrm_resources>
      </lrm>
    </node_state>
    <node_state id="node2" ha="active" uname="node2" crmd="online" join="member" expected="member" in_ccm="true">
    </node_state>
  </status>
</cib>
