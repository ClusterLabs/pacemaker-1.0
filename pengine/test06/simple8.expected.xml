<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="1" num_updates="1" dc-uuid="0" have-quorum="false" remote-tls-port="0" validate-with="pacemaker-1.0">
  <configuration>
    <crm_config><cluster_property_set id="cib-bootstrap-options"><nvpair id="nvpair" name="no-quorum-policy" value="ignore"/><nvpair id="nvpair" name="symmetric-cluster" value="true"/><nvpair id="nvpair" name="default-resource-stickiness" value="100"/></cluster_property_set></crm_config>
    <nodes>
      <node id="node1" uname="node1" type="member"/>
      <node id="node2" uname="node2" type="member"/>
    </nodes>
    <resources>
      <primitive id="rsc3" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc3.meta"/></primitive>
      <primitive id="rsc4" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc4.meta"/></primitive>
      <group id="foo"><meta_attributes id="group-foo.meta"/>
	<primitive id="rsc1" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc1.meta"/></primitive>
      </group>
      <group id="bar"><meta_attributes id="group-bar.meta"/>
	<primitive id="rsc2" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc2.meta"/></primitive>
      </group>
    </resources>
    <constraints>
    </constraints>
  </configuration>
  <status>
    <node_state id="node1" ha="active" uname="node1" crmd="online" join="member" expected="member" in_ccm="true">
      <lrm id="node1">
        <lrm_resources id="lrm_resources">
          <lrm_resource id="foo:rsc1" class="heartbeat" type="apache">
            <lrm_rsc_op id="foo-rsc1_start_0" operation="start" interval="0" op-status="0" rc-code="0" call-id="1" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" crm_feature_set="1.0.6" transition-magic=""/>
          </lrm_resource>
          <lrm_resource id="rsc3" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc3_start_0" operation="start" interval="0" op-status="0" rc-code="0" call-id="1" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" crm_feature_set="1.0.6" transition-magic=""/>
          </lrm_resource>
          <lrm_resource id="bar:rsc2" class="heartbeat" type="apache">
            <lrm_rsc_op id="bar-rsc2_start_0" operation="start" interval="0" op-status="0" rc-code="0" call-id="1" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" crm_feature_set="1.0.6" transition-magic=""/>
          </lrm_resource>
          <lrm_resource id="rsc4" class="heartbeat" type="apache">
            <lrm_rsc_op id="rsc4_start_0" operation="start" interval="0" op-status="0" rc-code="0" call-id="1" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8" crm_feature_set="1.0.6" transition-magic=""/>
          </lrm_resource>
        </lrm_resources>
      </lrm>
    </node_state>
    <node_state id="node2" ha="active" uname="node2" crmd="online" join="member" expected="member" in_ccm="true">
    </node_state>
  </status>
</cib>
